Resources/Prototypes/DeltaV/Entities/Structures/Machines/computers.yml

```diff
@@ -29,3 +29,51 @@
       state: request
     - map: ["computerLayerKeys"]
       state: tech_key
+
+- type: entity
+  abstract: true
+  parent: BaseComputer
+  id: BaseComputerDocking
+  components:
+  - type: Sprite
+    layers:
+    - map: ["computerLayerBody"]
+      state: computer
+    - map: ["computerLayerKeyboard"]
+      state: generic_keyboard
+    - map: ["computerLayerScreen"]
+      state: shuttle
+    - map: ["computerLayerKeys"]
+      state: generic_keys
+    - map: [ "enum.WiresVisualLayers.MaintenancePanel" ]
+      state: generic_panel_open
```

Comment:
spaces in inline lists everywhere, not just on the last one
`[ "computerLayerBody" ]`

Reply:
john copy paste